horizontalintegration/xmc-hztl-foundation-2024-q1
# Bug FIS-514 Country Selector UX

## What does this PR do and why?

This PR updates the CountrySelector component so that it matches the approved UX.

**NOTE:** The "Choose your country and language" text is populated via a dictionary item that I have not yet figured out how to publish so it may or may not appear.

## Screenshots or screen recordings

| Before | After |
| ------ | ----- |
| <img width="374" alt="Screenshot 2024-11-13 at 10 27 52 AM" src="https://github.com/user-attachments/assets/5668fea9-c1c8-4930-a23d-63a78f7a8c62"> | <img width="375" alt="Screenshot 2024-11-13 at 10 08 05 AM" src="https://github.com/user-attachments/assets/d45de107-278c-4dbe-97b9-7e8a33cb404d"> |

## How to set up and validate locally

1. Check out the branch _bug/FIS-514-country-selector_.
2. From a command prompt, navigate to _./src/hztl-foundation_ and execute `npm run start:connected`.
3. From a browser, navigate to http://localhost:3000. Confirm that the country selector in the header functions correctly and matches the approved UX at all breakpoints.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
